controller/composite/controller.go

```diff
@@ -240,23 +239,14 @@ func (pc *parentController) enqueueParentObject(obj interface{}) {
 }
 
 func (pc *parentController) updateParentObject(old, cur interface{}) {
-	// Ignore updates if the spec hasn't changed (e.g. our own status updates).
+	// Ignore updates where the ResourceVersion changed (not a resync)
+	// but the spec hasn't changed (e.g. our own status updates).
```

Comment:
but the `generation`

Reply:
It's more accurate to leave it as "spec hasn't changed". We can't always rely on `metadata.generation` to know if the spec changed, because not all resources have generation enabled.